test/torchaudio_unittest/functional/functional_cpu_test.py

```diff
@@ -179,3 +191,31 @@ def test_mask_along_axis_iid(self, mask_param, mask_value, axis):
 
         assert mask_specgrams.size() == specgrams.size()
         assert (num_masked_columns < mask_param).sum() == num_masked_columns.numel()
+
+
+@skipIfNoExec('sox')
+class ApplyCodecTestBase(TempDirMixin, TorchaudioTestCase):
```

Comment:
Can you rename this to `TestApplyCodec`? one thing, this is not a base class, the second, `TestXXX` is the common pattern used in torchaudio's test suite.